src/client.rs

```diff
@@ -65,17 +64,26 @@ impl Client {
     /// Create a connection to APNs using the provider client certificate which
     /// you obtain from your [Apple developer
     /// account](https://developer.apple.com/account/).
+    ///
+    /// Only works with the `openssl` feature.
+    #[allow(unused_variables)]
```

Comment:
Why?

Reply:
With removal of cfg_if and by making openssl the default one not needed anymore.